packages/sdks/core-js-sdk/src/httpClient/helpers/createFetchLogger.ts

```diff
@@ -97,28 +109,33 @@ const fetchWrapper =
     return resp;
   };
 
+// we should retry once in case we got these status codes:
+// 521: Web Server Is Down (Cloudflare error)
+// 524: A Timeout Occurred (Cloudflare error)
+const retryStatusCode = [521, 524];
```

Comment:
It should be defined before the actual use no?

Reply:
It's not that important in this case but I moved it